plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -59,6 +61,8 @@ export async function createRouter(
   const logger = parentLogger.child({ plugin: 'scaffolder' });
   const jobProcessor = new JobProcessor();
 
+  const discovery = SingleHostDiscovery.fromConfig(config);
+  const entityClient = new CatalogEntityClient({ discovery });
```

Comment:
What should I pass as ```config``` in tests, [here](https://github.com/spotify/backstage/blob/master/plugins/scaffolder-backend/src/service/router.test.ts#L100)?

Reply:
I think that you should either mock out this entity client in the tests so that the tests don't actually call out to the catalog service which wont exist when only starting the scaffolder, or use `msw` to mock the rest endpoint. I'd argue mocking the client is easier and better.